astroquery/vo_conesearch/tests/test_conesearch.py

```diff
@@ -104,8 +104,8 @@ def test_timeout(self):
         with pytest.raises(VOSError) as e:
             conesearch.conesearch(
                 SCS_CENTER, SCS_RADIUS, pedantic=self.pedantic, cache=False,
-                verbose=self.verbose, catalog_db=self.url, timeout=0.0001)
-        assert 'timed out' in str(e), 'test_timeout failed'
+                verbose=self.verbose, catalog_db=self.url, timeout=1e-6)
+        assert 'timed out' in str(e.value), 'test_timeout failed'
```

Comment:
@bsipocz , if you want `e.value` fix (needed for `pytest` 5) separately so it can be merged earlier, let me know.

Reply:
it doesn't yet causing issues, right? But either way, if you don't mind that can be a quick PR to be sure we are as good as possible.